1-exercises/B-todo-list/script.js

```diff
@@ -1,5 +1,22 @@
 function todoList(todos) {
   // Write your code here...
+  let content = document.querySelector("#content");
+  let unorderedList = document.createElement("ul");
+  content.appendChild(unorderedList);
+  todos.forEach((task) => {
```

Comment:
Nice! You wrote it in a DRY way. 

Please check this: https://codinglead.co/javascript/what-is-DRY-code#:~:text=There's%20a%20principle%20in%20programming,any%20change%20in%20one%20place.